jest.config.js

```diff
@@ -0,0 +1,13 @@
+const nextJest = require('next/jest');
```

Comment:
Can we do this like `jest.config.mjs`? To use ESM instead of CJS.

Reply:
Yes, the use of ESM should be documented

Reply:
Wdym?

Reply:
in our codebase we have no document specifying whether to use `cjs` or `esm`.

Reply:
Why would we? I'm just asking to use esm because our next config is also in esm.

Reply:
Oh oops we misunderstood each other

Reply:
No worries :)